src/components/currentLesson/CurrentLesson.tsx

```diff
@@ -0,0 +1,52 @@
+import './_currentLesson.scss';
+import { useEffect, useState } from "react"
+import { CurrentLessonProps } from "../../types/components/componentType"
+import defaultCoursePhoto from '../../img/defaultCoursePhoto.png';
+import axios from "axios";
+
+const CurrentLesson = () => {
+const [currentLessonProps, setCurrentLessonProps] = useState<CurrentLessonProps | null>(null);
+
+useEffect(() => {
+const fetchData = async () => {
+  try {
+    const response = await axios.get('URL/api/currentlesson');
+    const currentLessonData = response.data;
```

Comment:
If you use axios you must set settings for it. You can see it in file './src/services/requests.ts'. Also we will use services files for every request. So now you can delete or mock your GET request.

Reply:
fixed ?